src/cascadia/TerminalApp/TabHeaderControl.cpp

```diff
@@ -18,8 +18,18 @@ namespace winrt::TerminalApp::implementation
         // We'll only process the KeyUp event if we received an initial KeyDown event first.
         // Avoids issue immediately closing the tab rename when we see the enter KeyUp event that was
         // sent to the command palette to trigger the openTabRenamer action in the first place.
-        HeaderRenamerTextBox().KeyDown([&](auto&&, auto&&) {
+        HeaderRenamerTextBox().KeyDown([&](auto&&, auto e) {
             _receivedKeyDown = true;
+
+            // GH#9632 - mark navigation buttons as handled.
+            // This should prevent the tab view to use this key for navigation between tabs
```

Comment:
will this break ctrl-left (move back word), shift-left (select leftward), etc?

Reply:
No, this will not break it. The internal behavior remains unchanged (actually, since this even is bubbling up, it cannot physically affect the internal behavior). It just suggests the parent not to handle this event.